plugins/pagerduty/src/components/Escalation/EscalationPolicy.tsx

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright 2020 Spotify AB
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React from 'react';
+import { List, ListSubheader, LinearProgress } from '@material-ui/core';
+import { EscalationUsersEmptyState } from './EscalationUsersEmptyState';
+import { EscalationUser } from './EscalationUser';
+import { useAsync } from 'react-use';
+import { pagerDutyApiRef } from '../../api';
+import { useApi } from '@backstage/core';
+import { Alert } from '@material-ui/lab';
+
+type Props = {
+  policyId: string;
+};
+
+export const EscalationPolicy = ({ policyId }: Props) => {
+  const api = useApi(pagerDutyApiRef);
+
+  const { value: users, loading, error } = useAsync(async () => {
+    const oncalls = await api.getOnCallByPolicyId(policyId);
+    const users = oncalls.map(oncall => oncall.user);
+
+    return users;
+  });
+
+  if (error) {
+    return (
+      <Alert severity="error">
+        Error encountered while fetching information. {error.message}
+      </Alert>
+    );
+  }
+
+  if (loading) {
+    return <LinearProgress />;
+  }
+
+  return users!.length ? (
```

Comment:
This is OK but I'd prefer `if (!users?.length) { return empty state }` and then plain return the regular content below